### PR TITLE
refactor: Update old auth url

### PR DESCRIPTION
**Pull request by deep-volue**
- Updating old auth URLs based on task https://app.clickup.com/t/8693cpxcv
- Also corrected name of mergeable config

### CODE REVIEW

**Review by stigalvestad — COMMENTED** [review]
Almost there, you need to update the api url as well.

File: wapi/session.py [hunk 1]
```diff
@@ -18,7 +18,7 @@
 RETRY_DELAY = 0.5  # Delay between retried calls, in seconds.
 TIMEOUT = 300      # Default timeout for web calls, in seconds.
 API_URLBASE = 'https://api.wattsight.com'
```
Comment (artchyn) [inline wapi/session.py:20]: This as well?
```suggestion
API_URLBASE = 'https://api.volueinsight.com'
```

[the remark this answers is not in the document]
Comment (deep-volue) [reply wapi/session.py:20]: So, should I update all the API URLs as well ?

Comment (artchyn) [reply wapi/session.py:20]: What about Line:33,40,47,56 . Should we change it to Volue Insight?

File: sampleconfig.ini [hunk 2]
```diff
@@ -6,4 +6,4 @@ timeout = 300
 [OAuth]
 id = client_id
 secret = client_secret
-auth_urlbase = https://auth.wattsight.com
+auth_urlbase = https://auth.volueinsight.com
```
Comment (artchyn) [inline sampleconfig.ini:9]: what about L:2?

**Comment by deep-volue** [comment]
Updated API URLs as well

File: wapi/session.py [hunk 3]
```diff
@@ -17,8 +17,8 @@
 RETRY_COUNT = 4    # Number of times to retry
 RETRY_DELAY = 0.5  # Delay between retried calls, in seconds.
 TIMEOUT = 300      # Default timeout for web calls, in seconds.
-API_URLBASE = 'https://api.wattsight.com'
-AUTH_URLBASE = 'https://auth.wattsight.com'
+API_URLBASE = 'https://api.volueinsight.com'
+AUTH_URLBASE = 'https://auth.volueinsight.com'
 
 
 class ConfigException(Exception):
```
Comment (artchyn) [inline wapi/session.py:24]: Should we rename 'Wattsight API' as well to Volue Insight API? see L:33, etc.

File: wapi/session.py [hunk 4]
```diff
@@ -30,7 +30,7 @@ class MetadataException(Exception):
 
 
 class Session(object):
-    """ Establish a connection to Wattsight API
+    """ Establish a connection to Volue Insight API
```
Comment (artchyn) [inline wapi/session.py:33]: Almost there L:36 L:40 L:47 L:56, sorry.

Comment (deep-volue) [reply wapi/session.py:33]: Done all

**Review by artchyn — APPROVED** [review]

**Pull request merged**